src/components/layout/Undernames/Undernames.tsx

```diff
@@ -0,0 +1,188 @@
+    percentUndernamesLoaded,
+  ]);
+
+  function updatePage(page: number) {
+    setTablePage(page);
+  }
+
+  return (
+    <>
+      <div className="page">
+        <div className="flex-column">
+          <div className="flex flex-justify-between">
+            <div className="flex flex-row text-medium white bold">
+              <span className="flex faded text-medium bold">
+                <button
+                  className="faded text-medium bold underline link center"
+                  onClick={() => navigate('/manage/pdnts')}
+                >
+                  <ArrowLeft
+                    width={30}
+                    height={20}
+                    viewBox={'0 0 20 20'}
+                    fill={'var(--text-white)'}
+                  />
+                  Manage PDNTs
+                </button>
+                &nbsp;/&nbsp;
+                <Tooltip
+                  placement="top"
+                  title={id}
+                  showArrow={true}
+                  overlayStyle={{
+                    maxWidth: 'fit-content',
+                  }}
+                >
+                  <button
+                    className="faded text-medium bold underline link center"
+                    onClick={() =>
+                      navigate(`/manage/pdnts/${pdntId?.toString()}`)
+                    }
+                  >
+                    {pdntState?.name.length ? pdntState.name : '[PDNT]'}
+                  </button>
+                </Tooltip>
+                &nbsp;/&nbsp;
+                <span className="text-medium white">Manage Undernames</span>
+              </span>
+            </div>
+            {/* TODO add table breadcrumb */}
+            <div className="flex flex-row flex-right">
+              <button
+                disabled={undernameTableLoading}
+                className={
+                  undernameTableLoading
+                    ? 'outline-button center disabled-button'
+                    : 'outline-button center'
+                }
+                style={{
+                  padding: '0.75em',
+                }}
+                onClick={() => alert('implement add undername functionality')}
+              >
+                {/* TODO get undername logo from figma */}
+                {isMobile ? (
+                  <></>
+                ) : (
+                  <span
+                    className="text white"
+                    style={{ fontSize: '16px', padding: '0 0.2em' }}
+                  >
+                    Add Undername
+                  </span>
+                )}
+              </button>
+            </div>
+          </div>
+          {tableLoading ? (
+            <div
+              className="flex center"
+              style={{ paddingTop: '10%', justifyContent: 'center' }}
+            >
+              <Loader
+                message={`Loading undernames... ${
+                  percent ? Math.round(percent) : ''
+                }%`}
```

Comment:
this will say `Loading undernames... %`...either set percent to `0` on initial render, or don't add this postfix